### PR TITLE
fix(cli): Create config to parse the title back to imagery name

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: packages/shared/src/util.ts [hunk 1]
```diff
@@ -21,6 +21,20 @@ export function titleizeImageryName(name: string): string {
...
 
+/**
+ * Make a tile imagery title as imagery name
+ * @example
+ *  'Tasman rural 2018-19 0.3m' => 'tasman_rural_2018-19_0-3m'
+ */
+export function nameImageryTitle(title: string): string {
+  return title
+    .replace(/ /g, '_')
+    .replace('.', '-')
```
Comment (blacha) [inline packages/shared/src/util.ts:32]: this only does the first `.` might want to remove all `.` 's

File: packages/shared/src/util.ts [hunk 2]
```diff
@@ -21,6 +21,20 @@ export function titleizeImageryName(name: string): string {
...
+    .replace(/[^a-zA-Z0-9-_]/g, '')
```
Comment (blacha) [inline packages/shared/src/util.ts:33]: I wonder if there is a way to preserve the utf8 macrons here, eg `Manawatū`

Comment (Wentao-Kuang) [reply packages/shared/src/util.ts:33]: Not sure yet. I will make a test for this and see how to keep it. Might just working.

Comment (Wentao-Kuang) [reply packages/shared/src/util.ts]: Add `À-ȕ` in the filter to keep these macrons.

File: packages/shared/src/util.ts (hunk 1 above)
Comment (Wentao-Kuang) [reply packages/shared/src/util.ts]: All fixed, thank you.

File: packages/shared/src/util.ts (hunk 2 above)
Comment (blacha) [reply packages/shared/src/util.ts]: How far does that reach? will this will cover all macrons used in NZ?

Comment (Wentao-Kuang) [reply packages/shared/src/util.ts]: image
I think it covers all the `a e i o u` I think.

**Review by blacha — APPROVED** [review]

**Pull request merged**